README.md

```diff
@@ -72,7 +72,9 @@ For more information, see the [wiki](https://github.com/Azure/azure-storage-fuse
      * [OPTIONAL] **--ca-cert-file=/etc/ssl/certs/proxy.pem** : If external network is only available through a proxy server, this parameter should specify the proxy pem certificate otherwise blobfuse cannot connect to the storage account. This option is only available from version 1.3.7
      * [OPTIONAL] **--httpsproxy=http://10.1.22.4:8080/** : If external network is only available through a proxy server, this parameter should specify the proxy server along with the port which is 8080 unless there are some deviations from normal port allocation numbers. This option is only available from version 1.3.7
      * [OPTIONAL] **--httpproxy=http://10.1.22.4:8080/** : Only used when https is turned off using --use-https=false, and if external network is only available through a proxy server, this parameter should specify the proxy server along with the port which is 8080 unless there are some deviations from normal port allocation numbers. This option is only available from version 1.3.7
-     
+     * [OPTIONAL] **--max-retry=26** : In case of retryable failure from azure for any operation, number of maximum retries that blobfuse can do. 
+     * [OPTIONAL] **--max_retry_delay=60** : When a retry is attempted, maximum number of seconds allowed between two retries.
+     * [OPTIONAL] **--retry_delay_factor=1.2** : When exponential retry is attempted, factor in seconds by which retry interval grows exponentiallly.
```

Comment:
does the delay factor need to be configured?

Reply:
If we are making retry a configurable thing, then how much exponential factor we need to use shall be configurable. If not we can just remove this from README so that atleast if someone want a finer control they will have it.